fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -457,7 +443,7 @@ protected FHIROperationException buildUnsupportedResourceTypeException(String re
                             .value(Code.of("resource"))
                             .build())
                         .build())
-                .details(CodeableConcept.builder().text(string(Encode.forHtml(msg))).build())
+                .details(CodeableConcept.builder().text(string(msg)).build())
```

Comment:
I'd suggest a slight change...  on 437 Encode.forHtml(resourceTypeName) this is the part that's returning the reported problem.  Otherwise, I think this is a fine change.

Reply:
yeah, I think I like that.  the idea behind removing the Encode.forHtml here is that the EncodingVisitor will take care of all the encoding before we return it to the client, but given my other comment, I think I like the idea of backing that out AND making the change you suggest here (encoding only the part that is user-provided).